src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Data/Customizations/Documents/DocumentsOperationsExtensions.cs

```diff
@@ -66,15 +66,12 @@ public static class DocumentsOperationsExtensions
         /// </param>
         public static AutocompleteResult Autocomplete(
             this IDocumentsOperations operations, 
-            AutocompleteMode autocompleteMode,
-            string search,
+            string searchText,
             string suggesterName = null,
```

Comment:
Shouldn't this be required?

Reply:
The Autocomplete`Mode` parameter moved to AutocompleteParameters and is optional. It defaults to the oneTermMode by the service if not specified. There's one added test case that tests the default mode.

Reply:
Good catch. Thanks.